dozer-cli/src/live/state.rs

```diff
@@ -167,6 +167,13 @@ impl LiveState {
                 app_name: dozer.dozer.config.app_name.clone(),
                 connections,
                 endpoints,
+                enable_api_security: dozer
+                    .dozer
+                    .config
+                    .api
+                    .clone()
```

Comment:
No need to `clone`, `as_ref` should be enough. And this logic is different from above in that it doesn't use env var.

Reply:
updated using as_ref

Reply:
not really get it ??

Reply:
ah, i see. Since `live state` is for local running. User should only have api security from `dozer-config.yaml`.  Hardly the `DOZER_MASTER_TOKEN ` is manually set by user - only if they know about it - Only Cloud should use it when deploy. Anyway, i update it to ensure the synchronize logic